internal/payloads/sources_payload.go

```diff
@@ -31,9 +31,9 @@ func NewListSourcesResponse(sourceList []*clients.Source) []render.Renderer {
 }
 
 type SourceUploadInfoResponse struct {
-	Provider  string                     `json:"provider"`
-	AwsInfo   *clients.AccountDetailsAWS `json:"aws" nullable:"true"`
-	AzureInfo *clients.AzureSourceDetail `json:"azure" nullable:"true"`
+	Provider  string                       `json:"provider"`
+	AwsInfo   *clients.AccountDetailsAWS   `json:"aws" nullable:"true"`
+	AzureInfo *clients.AccountDetailsAzure `json:"azure" nullable:"true"`
 }
```

Comment:
Because the new cache key is based on generics and type infers the prefix, string build-in type cannot be used. So I introduced a special type here.